conda_build/build.py

```diff
@@ -3730,6 +3730,21 @@ def build_tree(
 
                         built_packages.update({pkg: dict_and_meta})
                 else:
+                    # warn about script_env if splitting with --no-test
+                    script_env = utils.ensure_list(
+                        metadata.get_value("build/script_env", [])
+                    )
+                    if script_env:
+                        env_name_list = ", ".join(
+                            [item.partition("=")[0] for item in script_env]
+                        )
+                        warnings.warn(
```

Comment:
Shouldn't this be `logging.Logger.warning` instead? 🤔

Reply:
I'm afraid I can't answer that. I didn't pick where the warning goes, I just moved the condition. If it should change, let me know or feel free to take over the PR with the change you would prefer.